Agent crashes when no pipeline is defined

**Describe the bug**
Agent crashes when adding infra meters pipeline with no explicit pipeline

**Steps to reproduce**
Add this to recources in default policy:

```yaml
    telemetry_collectors:
    - agent_group: default
      infra_meters:
        rabbitmq:
          per_agent_group: true
          receivers:
            rabbitmq:
              collection_interval: 1s
              endpoint: http://localhost:15672
              password: asdf
              username: otelu

```

**What did you expect to see?**
Agent trying to reach out to (nonexisting) rabbitmq server and printing some error logs.

**What did you see instead?**
crash

<details>

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0xc0020f7e98 addr=0xbd7be9c pc=0x832bcda]

goroutine 455 [running]:
github.com/fluxninja/aperture/pkg/panichandler.(*PanicHandlerRegistry).Crash(0xc000320090, {0xca42620, 0x13e0aa40})
	/src/pkg/panichandler/panichandler.go:95 +0xa7
github.com/fluxninja/aperture/pkg/panichandler.(*PanicHandlerRegistry).Go.func1.1()
	/src/pkg/panichandler/panichandler.go:105 +0x46
panic({0xca42620, 0x13e0aa40})
	/usr/local/go/src/runtime/panic.go:890 +0x263
github.com/fluxninja/aperture/pkg/otelcollector/infra-meter.addInfraMeter(0xc0020d42d0, {0xc001caf4f0, 0x4b}, 0xc002097180)
	/src/pkg/otelcollector/infra-meter/infra-meter.go:73 +0x9ba
github.com/fluxninja/aperture/pkg/otelcollector/infra-meter.AddInfraMeters(0xc0020d42d0, 0xc0020f9600)
	/src/pkg/otelcollector/infra-meter/infra-meter.go:36 +0x3fd
github.com/fluxninja/aperture/cmd/aperture-agent/agent.provideAgent.func1({{0xc001d9c67c, 0x42}, {0xc001c726c0, 0x10b, 0x120}, 0x1}, {0xfa216e0, 0xc001c6ecc0})
	/src/cmd/aperture-agent/agent/otel-config.go:91 +0x89e
github.com/fluxninja/aperture/pkg/notifiers.(*unmarshalKeyNotifier).Notify(0xc001d938f0, {{0xc001d9c67c, 0x42}, {0xc001c726c0, 0x10b, 0x120}, 0x1})
	/src/pkg/notifiers/unmarshal-notifier.go:54 +0x24f
github.com/fluxninja/aperture/pkg/notifiers.transformNotify({0xfa27ce0, 0xc001d938f0}, {{0xc001d9c67c, 0x42}, {0xc001c726c0, 0x10b, 0x120}, 0x1})
	/src/pkg/notifiers/notifier.go:118 +0x1e2
github.com/fluxninja/aperture/pkg/notifiers.(*keyTracker).addKeyNotifier(0xc001c17b80, {0xfa27ce0, 0xc001d938f0})
	/src/pkg/notifiers/tracker.go:66 +0x355
github.com/fluxninja/aperture/pkg/notifiers.(*DefaultTrackers).addKeyNotifier(0xc002412180, {0xfa27ce0, 0xc001d938f0})
	/src/pkg/notifiers/tracker.go:224 +0x255
github.com/fluxninja/aperture/pkg/notifiers.(*DefaultTrackers).addPrefixNotifier(0xc002412180, {0xfa21890, 0xc002405160})
	/src/pkg/notifiers/tracker.go:271 +0x416
github.com/fluxninja/aperture/pkg/notifiers.(*DefaultTrackers).Start.func1()
	/src/pkg/notifiers/tracker.go:396 +0x2ab
github.com/fluxninja/aperture/pkg/panichandler.(*PanicHandlerRegistry).Go.func1()
	/src/pkg/panichandler/panichandler.go:108 +0x7d
created by github.com/fluxninja/aperture/pkg/panichandler.(*PanicHandlerRegistry).Go
	/src/pkg/panichandler/panichandler.go:100 +0xd7
```

</details>

**What version did you use?**
dfc1dc732e7acb5df1d866695bb7d3132adbe528

**Environment**
playground

**Additional context**
https://docs.fluxninja.com/reference/policies/spec#infra-meter
> If there are no processors defined or only one processor is defined, the pipeline definition can be omitted. In such cases, the pipeline will automatically use all given receivers and the defined processor (if any). However, if there are more than one processor, the pipeline must be defined explicitly.

I recall this feature was working before 2.0.0 (before https://github.com/fluxninja/aperture/pull/1890)

--- a/pkg/otelcollector/infra-meter/infra-meter.go
+++ b/pkg/otelcollector/infra-meter/infra-meter.go
@@ -70,6 +70,12 @@ func addInfraMeter(
 		config.AddProcessor(id, cfg)
 	}
 
+	if infraMeter.Pipeline == nil {
+		// We treat empty pipeline the same way as not-set pipeline, normalize.
+		// This also allows to avoid nil checks below.
+		infraMeter.Pipeline = &policylangv1.InfraMeter_MetricsPipeline{}
+	}
+
 	if len(infraMeter.Pipeline.Receivers) == 0 && len(infraMeter.Pipeline.Processors) == 0 {
 		if len(infraMeter.Processors) >= 1 {
 			return fmt.Errorf("empty pipeline, inferring pipeline is supported only with 0 or 1 processors")

